intelmq/tests/bots/parsers/shadowserver/testdata/scan_adb.csv

```diff
@@ -1,3 +1,3 @@
-"timestamp","ip","protocol","port","hostname","tag","asn","geo","region","city","naics","sic","name","model","device","features"
+"timestamp","ip","protocol","port","hostname","tag","asn","geo","region","city","naics","sic","name","model","device","features","device_vendor","device_type","device_model","device_version","device_sector"
```

Comment:
as above: looks like the data is missing

Reply:
Fixed